content/en/account_management/org_settings.md

```diff
@@ -13,42 +13,42 @@ Read the [user management][2] documentation to add, edit, and disable users.
 
 #### Service accounts
 
-Service accounts are non-human users that can have roles and own application keys. You can use service accounts to access Datadog APIs without having your application or script be associated with a particular person. Service account application keys are one-time-view only, and are not retrievable after they've been created.
+[Service accounts][3] are non-human users that can have roles and own application keys. You can use service accounts to access Datadog APIs without having your application or script be associated with a particular person. Service account application keys are one-time-view only, and are not retrievable after they've been created.
```

Comment:
I would align the copy here with the same headline as the service accounts page. Sounds weird here that we call a service accounts a non human users. Maybe replace it with "Service accounts are non-interactive accounts that can .."